packages/pretzelai-extension/src/PretzelInlineProvider.ts

```diff
@@ -119,7 +121,6 @@ export class PretzelInlineProvider implements IInlineCompletionProvider {
     return false;
   }
 
-  public isFetching: boolean = false;
```

Comment:
I removed this variable. Seems unused? Can you verify I didn't miss anything @prasoon2211 ?